masterlist.yaml

```diff
@@ -898,6 +898,17 @@ plugins:
         itm: 385
         udr: 93
         nav: 3
+      # v1.6.1170.0
+      - <<: *reqManualFix
```

Comment:
While technically correct, for the _vanilla game files_ we manually adjust the reports to say `*quickClean` instead of `*reqManualFix` so this needs to be changed.